analysis/experiment_results.py

```diff
@@ -42,6 +42,13 @@ def __init__(self,
             # Take name from first row.
             self.name = experiment_df.experiment[0]
 
+        # FuzzBench repo commit hash.
+        self.git_hash = None
+        if 'git_hash' in experiment_df.columns:
```

Comment:
Do we need this if, i think it is better to exception here since this column is always supposed to here.

Reply:
Yes, we need this to support the use case of [generating reports](https://google.github.io/fuzzbench/developing-fuzzbench/custom_analysis_and_reports#generating-reports) from previously published data.csv.gz-s, as they don't have a git_hash column. Another option is to regenerate the [previous reports](https://www.fuzzbench.com/reports/index.html) and the raw data files.

Reply:
LGTM then, forgot about that. Thanks for verifying.